webknossos/examples/learned_segmenter.py

```diff
@@ -75,10 +76,18 @@ def main() -> None:
     segmentation_layer.add_mag(mag, compress=True).write(segmentation)
 
     # Get your auth token from https://webknossos.org/auth/token
-    with wk.webknossos_context(
-        url="http://localhost:9000", token="secretSampleUserToken"
-    ):
-        url = dataset.upload()
+    with wk.webknossos_context(url="http://localhost:9000", token="secretSampleUserToken"):
+        url = dataset.upload(
+            layers_to_link=[
+                LayerToLink(
+                    annotation.dataset_name,
+                    "color",
+                    "color",
+                    "scalable_minds",
```

Comment:
The `new_layer_name` is superfluous I think:

```suggestion
                    organization_name="scalable_minds",
                    dataset_name=annotation.dataset_name,
                    layer_name="color",
```